pkgs/development/python-modules/opensfm/default.nix

```diff
@@ -44,19 +44,15 @@ let
 in
 buildPythonPackage rec {
   pname = "OpenSfM";
-  version = "0.5.2";
+  version = "0.5.2pre";
```

Comment:
Please don't create arbitrary versions. If this is some commit not tagged use the unstable format.

Reply:
What is the unstable format?

Reply:
tldr: I wanted to make it explicit that the version hasn't been released yet

Reply:
> What is the unstable format?

`unstable-XXXX-XX-XX` where XXXX-XX-XX is the date of the commit.

Reply:
✅